php-frontend/src/main/java/org/sonar/php/utils/LiteralUtils.java

```diff
@@ -33,4 +36,146 @@ public static long longLiteralValue(String literalValue) {
     return Long.decode(value);
   }
 
+  // https://www.php.net/manual/en/language.types.string.php
+  public static String stringLiteralValue(String literalValue) {
+    return StringLiteralParser.stringValue(literalValue);
+  }
+
+  private abstract static class StringLiteralParser {
+
+    private final String valueWithoutDelimiters;
+
+    static String stringValue(String literalValue) {
+      String valueWithoutDelimiters = literalValue.substring(1, literalValue.length() - 1);
```

Comment:
I'm a bit confused by the word `delimiter`. Maybe `valueWithoutQuates` will have a better fit.